tests/test_hypersphere.py

```diff
@@ -140,23 +174,100 @@ def test_exp_vectorization(self):
         one_tangent_vec = self.space.projection_to_tangent_space(
             one_vec, base_point=one_base_point)
         result = self.metric.exp(one_tangent_vec, one_base_point)
-        self.assertTrue(gs.allclose(result.shape, (1, dim + 1)))
+        self.assertTrue(
+            gs.isclose(result.shape, dim + 1),
+            'result.shape = {}'.format(result.shape))
```

Comment:
I feel that yoouv'e been using this pattern a fair amount - is there a way you could refactor this (not necessarily in this PR) in a test_util file or somtg?

Reply:
Yes, good idea.